contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -153,11 +161,8 @@ contract UniswapAnchoredView is AggregatorValidatorInterface, UniswapConfig, Own
             anchorPrice = fetchAnchorPrice(config.symbolHash, config, ethPrice);
         }
 
-        if (reporterInvalidated[msg.sender]) {
-            prices[config.symbolHash] = anchorPrice;
-            emit PriceUpdated(config.symbolHash, anchorPrice);
-        } else if (isWithinAnchor(reporterPrice, anchorPrice)) {
-            prices[config.symbolHash] = reporterPrice;
+        if (isWithinAnchor(reporterPrice, anchorPrice)) {
```

Comment:
This is probably fine, but I want to point out that this will mean that we `emit PriceUpdated` even while the failover is active.

Reply:
I think that is OK.

If failover was activated due to stale aggregator, these events firing again would signal that it is safe to `deactivateFailover` and roll back to the anchored prices.